.github/workflows/dev_on_workflow_web_proxy_deploy.yml

```diff
@@ -47,9 +50,15 @@ jobs:
         with:
           secrets: |-
             SENTRY_AUTH_TOKEN:passculture-metier-ehp/passculture-app-native-sentry-token
+      - name: Switch to google prod account for prod
+        if: ${{ inputs.ENV }} == 'production'
```

Comment:
pourquoi on n'ajoute pas un `if: ${{ inputs.ENV }} != 'production'` sur `name: Authentification to Google` ?

Reply:
car jai quand meme besoin de recup'le secret sentry en amont

Reply:
on se sert de EHP comme d'un store de secret global, ce qui évite d'avoir à les chercher dans chaque env, Il faut encore qu'on rajoute des labels pour clarifier la chose mais c'est bien plus pratique